crates/sui-indexer/src/handlers/checkpoint_handler.rs

```diff
@@ -130,6 +130,9 @@ where
                 "Checkpoint {} committed with {tx_count} transactions and {object_count} object changes.",
                 next_cursor_sequence_number
             );
+            self.metrics
+                .transaction_per_checkpoint
+                .observe(tx_count as f64);
```

Comment:
why is count an f64?

Reply:
this is the type Grafana histogram needs.